<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TotalPaymentsDialog</class>
 <widget class="QDialog" name="TotalPaymentsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="totalPaymentsLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="productNameTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>104</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="productPriceTextEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>104</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="productQuantityTextEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>104</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
